### ISSUE TITLE
Selector not displaying full language names

### ISSUE THREAD

**Issue by AllienWorks** [splitbrain/dokuwiki-plugin-translation#253]
I'm using Dokuwiki with Translation plugin and Bootstrap3 template (besides other extensions). I have two instances of this Dokuwiki setup running – one couple years old and a new one to replace it soon.

Regarding Translation plugin, both Dokuwiki instances share the exact configuration (double-checked in `/conf/local.php`), but the new one doesn't display full language names in language selector, no matter what I do and what settings I change.

To illustrate:

![dokuwiki-translation-issue](https://user-images.githubusercontent.com/1965795/60526239-35c99c00-9cf0-11e9-9c55-3ccc76f82eff.png)

- both instances are running **2018-04-22b "Greebo"**
- both are on the same webhost (not that it should matter)
- both use this settings:
```
$conf['plugin']['translation']['translations'] = 'fr, de, nl, es, zh, ko, ru';
$conf['plugin']['translation']['display'] = 'name,flag,title';
$conf['plugin']['translation']['checkage'] = 1;
$conf['plugin']['translation']['copytrans'] = 1;
```
- both display the language selector via Bootstrap3's nav bar (which uses a slightly customized theme, but only color-related, nothing functional was tweaked)

Any ideas what might cause this? I feel like I've tried everything already, twice. Getting a bit desperate here :)

**Comment by splitbrain** [comment]
🤔 and both are the same version of the translation plugin? that's very odd indeed. Maybe there's something cached. try resaving the configuration.

**Comment by AllienWorks** [comment]
> thinking and both are the same version of the translation plugin?

Tbh I don't know where to get plugin's version, but neither of the instances report that it's updatable. So I'd say they're both on the latest version.

> Maybe there's something cached. try resaving the configuration.

Oh, I've purged the cache like a million times already (resaved configuration many times as well).

I guess it must be one of the other extensions I have installed. It's true that the new one uses more extensions than the old one.. Well thanks anyway, for your super fast reply! :)
